.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "eslint.enable": true,
+    "eslint.validate": [
+        { "language": "javascript", "autoFix": true },
+        { "language": "vue", "autoFix": true },
+        { "language": "typescript", "autoFix": true }
+    ],
+    "eslint.options": {
+        "extensions": [".ts", ".js"]
+    },
+    "vetur.validation.template": false,
+    "eslint.autoFixOnSave": true
+}
```

Comment:
not everyone use vs code, it should not depend on the use of vs code

Reply:
settings.json is just a convenience helper for people who use vscode. The linter does the same job in the end.